src/rollback_to_stable/rts.c

```diff
@@ -87,12 +87,12 @@ __rts_progress_msg(WT_SESSION_IMPL *session, struct timespec rollback_start,
     /* Time since the rollback started. */
     time_diff = WT_TIMEDIFF_SEC(cur_time, rollback_start);
 
-    if ((time_diff / WT_PROGRESS_MSG_PERIOD) > *rollback_msg_count) {
+    if ((time_diff / WT_MINUTE) > *rollback_msg_count) {
```

Comment:
What's the reason to go from 20 seconds to 60 seconds here?

Reply:
My idea was that with the new included progress message creating a message every 20 seconds, we can make this message less frequent, to reduce over flooding of log messages.

Reply:
That's reasonable, but I don't think an extra log line every 20 seconds is going to be an issue during RTS. I'd also prefer to use the same `WT_PROGRESS_MSG_PERIOD` macro, it'd be confusing to adjust that and not have this message change frequency.

Reply:
ok reverted back [245e6cf](https://github.com/wiredtiger/wiredtiger/pull/9194/commits/245e6cfa0efdda4e418b2f774ced4945332a22e3)

Reply:
oops nice catch